src/ui/public/notify/lib/_format_msg.js

```diff
@@ -25,7 +25,12 @@ export function formatMsg(err, from) {
     rtn += formatMsg.describeError(err);
   } else if (has(err, 'status') && has(err, 'data')) {
     // is an Angular $http "error object"
-    rtn += 'Error ' + err.status + ' ' + err.statusText + ': ' + err.data.message;
+    if (err.status === -1) {
+      // status = -1 indicates that the request was failed to reach the server
+      rtn += 'Connection failed! Is Kibana running?';
```

Comment:
I like it, but I'm going to drop the ` to the Internet` part at the end to avoid confusing users running it locally / internally on an intranet.